Source/Core/Core/AchievementManager.h

```diff
@@ -188,6 +199,7 @@ class AchievementManager
   Common::WorkQueueThread<std::function<void()>> m_queue;
   Common::WorkQueueThread<std::function<void()>> m_image_queue;
   std::recursive_mutex m_lock;
+  std::recursive_mutex m_video_lock;
```

Comment:
There's no need for this lock as far as I can tell, all usages are already locked by `m_lock`.